ibis/expr/datatypes.py

```diff
@@ -83,7 +83,9 @@ def __repr__(self) -> str:
         )
 
     def __str__(self) -> str:
-        return self.name.lower()
+        return '{}{}'.format(
```

Comment:
can you avoid putting PRs on top of other ones, unless its *really* clear. Smaller is *much* better.

Reply:
I added this because it was really helping checking some fixes I made. next time I will make sure to point clearly that. thanks for the feedback.